### ISSUE TITLE
Qt5 version does not enable dbus phonon kwallet support properly

### ISSUE THREAD

**Issue by cheese** [qterm/qterm#10]
Qt5 versions of CMake variables are not tested.

**Comment by hephooey** [comment]
Yes, I am aware of that. I am not sure if it is wise to enable kwallet support for Qt5 without testing on KWallet5, which I do not have installed at this moment. We might also explore more general possibilities like qtkeychain. Patches are always welcome though :)
